src/css/molecules/card.scss

```diff
@@ -1,18 +1,85 @@
+.section-header,
 .card-container {
-  display: grid;
-  grid-gap: $grid-gap-mobile;
-  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0;
+  position: relative;
 
   @media #{$mq-md} {
-    grid-column-gap: $grid-gap;
-    grid-row-gap: 24px;
+    max-width: 1024px;
+  }
+}
+
+.card-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  .card {
+    max-width: 300px;
+    margin-right: $grid-gap;
+  }
+
+  .card-featured {
+    max-width: 494px;
+    margin-right: $grid-gap;
+  }
+
+  .card-link {
+    flex-grow: 1;
+    margin-bottom: 24px;
   }
 }
 
-.card-container-2-wide {
-  grid-template-columns: repeat(auto-fill, minmax(340px, 1fr));
+.filter-list-sidebar {
+  margin-right: $grid-gap;
+}
+
+.item-topic-link {
+  max-width: 320px;
+  margin-bottom: 24px;
 }
 
+@supports (grid-gap: $grid-gap-mobile) {
```

Comment:
Can we just check for `display: grid` here?

Reply:
Sorry yes, I thought IE supported grid, buts its -ms prefixed.